src/utils/error.ts

```diff
@@ -16,14 +16,20 @@ enum ErrorTypes {
   changePasswordValidation = 'CHANGE_PASSWORD_VALIDATION_ERROR',
   confirmationValidation = 'CONFIRMATION_VALIDATION_ERROR',
   emailAlreadyExists = 'EMAIL_ALREADY_EXISTS',
+  emptyArgs = 'EMPTY_ARGS',
   forgotPasswordValidation = 'FORGOT_PASSWORD_VALIDATION_ERROR',
   googleLoginValidation = 'GOOGLE_LOGIN_VALIDATION_ERROR',
   incorrectPasword = 'INCORRECT_PASSWORD',
   loginValidation = 'LOGIN_VALIDATION_ERROR',
   passwordLinkExpired = 'PASSWORD_LINK_EXPIRED',
   registrationValidation = 'REGISTRATION_VALIDATION_ERROR',
   resendConfirmationValidation = 'RESEND_CONFIRMATION_VALIDATION_ERROR',
+  sheetNotFound = 'SHEET_NOT_FOUND',
```

Comment:
Maybe split `ErrorTypes` between multiple service related error types. You would have multiple enums (for example: `AuthErrorTypes`, `UserErrorTypes` and `SheetErrorTypes`) that you would merge in a single object type:

```typescript
enum AuthErrorTypes { ... }
enum SheetErrorTypes { ... }
enum UserErrorTypes { ... }

const ErrorTypes = { ...AuthErrorTypes, ...SheetErrorTypes, ...UserErrorTypes }

type ErrorTypes = typeof ErrorTypes
```